Content.YAMLLinter/Program.cs

```diff
@@ -103,9 +103,18 @@ private static async Task<int> Main(string[] _)
             return (yamlErrors, fieldErrors);
         }
 
-        public static async Task<(Dictionary<string, HashSet<ErrorNode>> YamlErrors , List<string> FieldErrors)>
+        public static async Task<(Dictionary<string, HashSet<ErrorNode>> YamlErrors, List<string> FieldErrors)>
             RunValidation()
         {
+            var assemblies = AppDomain.CurrentDomain.GetAssemblies();
```

Comment:
This gets the assemblies loaded by the current application, which in this case is the YAML Linter. This includes Server, Shared, and Client assemblies for Content, but seems to skip Robust.Client, which I guess is never used in the Linter?

This doesn't seem to cause any problems, but I'm not sure if that's just lucky... if a new prototype used a component from Robust.Client, I'm not sure if that would cause the linter to load that assembly (which would be great) or not (which would cause problems). It might be better to explicitly name the assemblies to load, but I'm not sure.

If anyone more familiar with assembly loading could comment, it would be much appreciated.